gitapp/lib/users_list/api_repository.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:gitapp/users_list/user_model.dart';
+import "package:http/http.dart" as http;
+import 'dart:convert' as convert;
+
+class apiRepository {
+  apiRepository();
+
+  static final apiRepository apirep = new apiRepository();
+
+  Future<List<User>> fetchUsers(String username) async {
+    List<User> listofUsers = List.empty(growable: true);
+    String httpaddress =
+        'https://api.github.com/search/users?q=$username&per_page=100';
```

Comment:
In here we do not support paging. And it could be done like this. So when we shall scroll to the last element of the list, then we could ask for another page of the documents.

Reply:
it will be in the next pull request